### PR TITLE
Fixes #149 - Make it more clear Privacy Budget is a proposal

### LINKED ISSUE #149: Explainer should not suggest that the controversial "Privacy Budget" proposal will solve fingerprinting
The [Spam filtering and bot detection] section of the explainer suggests that Privacy Budget will be a solution to fingerprinting. At this time Privacy Budget is an experimental Chrome proposal that is not even in incubation yet, and which has drawn criticism from other browser vendors. It would probably be better to avoid taking a stance on this controversy, even though the explainer is not normative.

### PR DESCRIPTION
**Pull request by miketaylr**
...in the explainer.

PTAL @amtunlimited

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -454,18 +454,19 @@ This is a case of fingerprinting that is not user-hostile, and therefore one we
 would like to preserve.  With UA-CH this will be initially enabled by active
 collection of the various hints.
 
-This will work until something like [Privacy
-Budget](https://github.com/bslassey/privacy-budget) will prevent sites from
-collecting user identifying levels of entropy about their users.  By then, one
-can hope that alternative methods will exist for spam filtering that will
-obsolete the use of fingerprinting for that purpose.
+We hope that alternative methods or APIs will exist in the future to address the
```
Comment (amtunlimited) [inline README.md:457]: Two "in the futures".

**Review by amtunlimited — APPROVED** [review]
LGTM % nit

File: README.md [hunk 2]
```diff
@@ -454,18 +454,19 @@ This is a case of fingerprinting that is not user-hostile, and therefore one we
...
+spam filtering and bot detection use cases in the future, as browsers may decide
```
Comment (yoavweiss) [inline README.md:458]: spurious "in the future" :)

File: README.md (hunk 1 above)
Comment (miketaylr) [reply README.md:457]: That's so you know it's "future-future"

(thanks)

File: README.md (hunk 2 above)
Comment (miketaylr) [reply README.md:458]: heh, thanks

**Pull request merged**